ld/devices.data

```diff
@@ -451,9 +451,9 @@ lpc43xx_m4 lpc43xx CPU=cortex-m4 FPU=hard-fp4-sp-d16
 # the STM32 families
 
 stm32f0 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m0 FPU=soft
-stm32f1 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m3 FPU=soft
-stm32f2 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m3 FPU=soft
-stm32f3 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m4 FPU=hard-fpv4-sp-d16
+stm32f1 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m1 FPU=soft
+stm32f2 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m2 FPU=soft
+stm32f3 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m3 FPU=hard-fpv4-sp-d16
```

Comment:
no.  please elaborate what this is meant to be fixing.  There is no such thing as a cortex-m2, and cortex-m1 is the fpga implementation.

Reply:
You're right. I thought it fixed some issues for me but it didn't. Correlation not causation...